lib/hand.rb

```diff
@@ -76,11 +54,23 @@ def suit_occurences
   end
 
   def card_sum
-    @cards.collect { |card| card.rank(Deck.FACE_RANKS)}
+    @cards.collect { |card| card.rank(Deck::VALUES)}
           .inject { |sum, card| sum + card}
   end
-  
+
   def all_unique?
-    card_occurences.count { |k,v| v == 1} == 5
+    card_occurences.count { |_k, v| v == 1 } == 5
```

Comment:
How about comparing the count to the length of the cards array instead of the magic number? Might be a little more explicit.